emission/core/wrapper/tour_model.py

```diff
@@ -0,0 +1,14 @@
+import logging
+import emission.core.wrapper.wrapperbase as ecwb
+
+class TourModel(ecwb.WrapperBase):
```

Comment:
I am not sure if TourModel needs to be a wrapper class in its current form. Maybe you need to put the common places and common trips into it?